src/doc/rustc/src/platform-support/riscv64gc-unknown-linux-gnu.md

```diff
@@ -0,0 +1,127 @@
+# `riscv64gc-unknown-linux-gnu`
+
+**Tier: 2 (with Host Tools)**
+
+RISC-V targets using the *RV64I* base instruction set with the *G* collection of extensions, as well as the *C* extension.
+
+
+## Target maintainers
+
+-  TODO
```

Comment:
I see [rust-embedded](https://github.com/rust-embedded/wg#the-risc-v-team) specifies some people that are part of the RISC-V team. I am not sure if this is the same as the target maintainers. If not, maybe they would know who this should be.

CC: @almindor @dkhayes117 @romancardenas @MabezDev @jessebraham

Reply:
`riscv` is a generic group, for me for example, while I'm happy to look at PRs, 64bit risc is something I cannot test since I currently do not own such hardware. Same goes for `-linux` targets, I only have bare-metal 32bit MCUs.

Reply:
Hi folks. As discussed separated with @Hoverbear, I am happy to help. @Hoverbear: Please let me know if there is anything else needed from me. Also note that there are at least 2 more colleagues who will shortly chime in in this thread who have agreed to be a part of this maintainers group. Thanks!

Reply:
Please feel free to include me as part of this maintainers group :)

Reply:
Count me in here, as a long-time RISC-V toolchain contributor, I am happy to help triage and resolve toolchain compiler issue for RISC-V :)

Reply:
Perfect!

Reply:
Count me in too.